src/main/java/org/junit/internal/runners/statements/RunBefores.java

```diff
@@ -20,6 +31,9 @@ public RunBefores(Statement next, List<FrameworkMethod> befores, Object target)
 
     @Override
     public void evaluate() throws Throwable {
+        if (null != previous) {
+            previous.evaluate();
```

Comment:
A `RunIgnores` class sounds reasonable. Does that mean the changes to `RunBefores` can be reverted?